### DISCUSSION TITLE
Different Attributes used in Artillery

### DISCUSSION THREAD

**Discussion by kshamajk** [artilleryio/artillery#1000] [Artillery]
Hi @hassy 

I am new to Artillery. I wanted to  know if is there any link/document with respect to different attributes used in Artillery.
i.e, Attributes like flow, log, phases, payload etc.

**Comment by hassy** [comment]
hi @kshamajk 👋 welcome to Artillery!

This page is a good place to start to see the top level options available:

https://artillery.io/docs/guides/guides/test-script-reference.html

After that, engine-specific pages are the best option for learning more, e.g. if you're going to be testing HTTP-based services:

https://artillery.io/docs/guides/guides/http-reference.html

We also have a repo with examples which is being updated frequently:

https://github.com/artilleryio/artillery-examples

If there's something you'd like to see, or if anything is unclear, let us know!
